source/extensions/quic_listeners/quiche/BUILD

```diff
@@ -459,14 +459,15 @@ envoy_cc_library(
     ],
 )
 
-envoy_cc_library(
+envoy_cc_extension(
     name = "udp_gso_batch_writer_config_lib",
     srcs = ["udp_gso_batch_writer_config.cc"],
     hdrs = ["udp_gso_batch_writer_config.h"],
+    category = "envoy.udp_packet_writers",
+    security_posture = "unknown",
+    status = "alpha",
     tags = ["nofips"],
-    visibility = [
-        "//test/server:__subpackages__",
-    ],
+    visibility = ["//visibility:public"],
```

Comment:
do we need to leave this visibility locked down until we have bazel compile out code, or do we trust CI to catch leakage of folks linking to it?

Reply:
I think this is only needed for some docs stuff, so if I hide the GRO extension for now I may be able to revert this. Let me check.